pkg/cache/cache.go

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Bartłomiej Płotka @bwplotka
+// Licensed under the Apache License 2.0.
+
+package cache
```

Comment:
It would be amazing to get some feedback to their project (just mention, don't expect anything).